### PR TITLE
Fix #1808: check for None in description

### LINKED ISSUE #1808: Error to ingest metadata from Glue
Hey Guys,

We are trying to deploy 0.7.0 in our sandbox env and we had some errors trying to get metadata from Glue:

[2021-12-16 19:00:48,167] {pod_launcher.py:149} INFO - 2021-12-16 19:00:48,166 - ERROR - Creating new ModelMetaclass for name/glue_metastore. Error 404
[2021-12-16 19:00:49,809] {pod_launcher.py:149} INFO - Traceback (most recent call last):
[2021-12-16 19:00:49,809] {pod_launcher.py:149} INFO -   File "/opt/metadata_ingestion/main.py", line 120, in <module>
[2021-12-16 19:00:49,810] {pod_launcher.py:149} INFO -     metadata_ingestion_workflow(config=ingestion_config_function)
[2021-12-16 19:00:49,810] {pod_launcher.py:149} INFO -   File "/opt/metadata_ingestion/main.py", line 111, in metadata_ingestion_workflow
[2021-12-16 19:00:49,810] {pod_launcher.py:149} INFO -     workflow.execute()
[2021-12-16 19:00:49,810] {pod_launcher.py:149} INFO -   File "/opt/venv/lib/python3.9/site-packages/metadata/ingestion/api/workflow.py", line 163, in execute
[2021-12-16 19:00:49,811] {pod_launcher.py:149} INFO -     self.sink.write_record(processed_record)
[2021-12-16 19:00:49,812] {pod_launcher.py:149} INFO -   File "/opt/venv/lib/python3.9/site-packages/metadata/ingestion/sink/metadata_rest.py", line 113, in write_record
[2021-12-16 19:00:49,812] {pod_launcher.py:149} INFO -     self.write_tables(record)
[2021-12-16 19:00:49,812] {pod_launcher.py:149} INFO -   File "/opt/venv/lib/python3.9/site-packages/metadata/ingestion/sink/metadata_rest.py", line 164, in write_tables
[2021-12-16 19:00:49,812] {pod_launcher.py:149} INFO -     description=db_and_table.location.description.strip(),
[2021-12-16 19:00:49,812] {pod_launcher.py:149} INFO - AttributeError: 'NoneType' object has no attribute 'strip'
[2021-12-16 19:00:51,842] {pod_launcher.py:198} INFO - Event: openmetadata-ingestion-dag-get-glue-metadata.7c9a21dc2e5b497cbb88a1a6191a565d had an event of type Failed
[2021-12-16 19:00:51,842] {pod_launcher.py:308} ERROR - Event with job id openmetadata-ingestion-dag-get-glue-metadata.7c9a21dc2e5b497cbb88a1a6191a565d Failed
[2021-12-16 19:00:51,849] {pod_launcher.py:198} INFO - Event: openmetadata-ingestion-dag-get-glue-metadata.7c9a21dc2e5b497cbb88a1a6191a565d had an event of type Failed
[2021-12-16 19:00:51,849] {pod_launcher.py:308} ERROR - Event with job id openmetadata-ingestion-dag-get-glue-metadata.7c9a21dc2e5b497cbb88a1a6191a565d Failed
[2021-12-16 19:00:51,911] {taskinstance.py:1463} ERROR - Task failed with exception

Version: openmetadata-ingestion==0.5.0rc1

Thank you

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/src/metadata/ingestion/sink/metadata_rest.py [hunk 1]
```diff
@@ -159,9 +162,13 @@ def write_tables(self, db_and_table: OMetaDatabaseAndTable):
 
             created_table = self.metadata.create_or_update(table_request)
             if db_and_table.location is not None:
+                if db_and_table.location.description is not None:
+                    db_and_table.location.description = (
+                        db_and_table.location.description.strip()
+                    )
```
Comment (amiorin) [inline ingestion/src/metadata/ingestion/sink/metadata_rest.py:168]: Why is it between parentheses

Comment (harshach) [reply ingestion/src/metadata/ingestion/sink/metadata_rest.py:168]: its the code formatter moving this into next line

Comment (harshach) [reply ingestion/src/metadata/ingestion/sink/metadata_rest.py:168]: image

Comment (amiorin) [reply ingestion/src/metadata/ingestion/sink/metadata_rest.py:168]: 👍

**Review by amiorin — APPROVED** [review]

**Pull request merged**